lib/atomic-management.js

```diff
@@ -92,26 +90,24 @@ export default {
     this.statusBar = statusBar;
     this.tooltips = atom.tooltips;
     this.tile = this.statusBar.addLeftTile({item: this.element, priority: 100});
-    function sleep(ms) {
-        return new Promise(resolve => setTimeout(resolve, ms));
-    }
-    async function initTools() {
-      await sleep(1000);
-      ReactDom.render(<RootController
-        statusBar={this.statusBar}
-        tooltips={this.tooltips}
-        projectConfigs={this.projectConfigs}
-        enabledConfigs={this.enabledProjects}
-        onReorder={this.onReorder.bind(this)}
-        onCheck={this.onCheck.bind(this)}
-        />, this.element, this.startUp.bind(this));
-    }
-    initTools.bind(this)();
+    this.themelistener = atom.themes.onDidChangeActiveThemes(() => {
```

Comment:
I'm sure you tested this, but are you sure this doesn't create a NEW status bar every time the themes change?